### DISCUSSION TITLE
thanhle.blog/nhung-dieu-khong-ai-day-ban-ve-nganh-phan-mem-p1

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#79] [Ideas]
# thanhle.blog/nhung-dieu-khong-ai-day-ban-ve-nganh-phan-mem-p1

Sự thật khó nhai về ngành này mà mấy bạn “chẻ” nên biết

https://thanhle.blog/blog/nhung-dieu-khong-ai-day-ban-ve-nganh-phan-mem-p1

**Comment by um1chc5** [comment]
Bài viết rất hay. Cho em hỏi anh dùng cách gì để pull được nội dung từ notion sang blog của mình ạ, em cảm ơn.

**Reply by thanhlmm** [reply]
Mình có dùng một Open source để query data từ Notion thành API á. 

Bạn có thể check detail blog mình ở https://github.com/thanhlmm/blog/blob/master/api-notion/README.md

**Comment by chuongregister** [comment]
Bài viết hay, mình thấy từ nguồn TopDev. Mình là Dev thôi những cũng có nhiều suy tư trong bài viết. Cũng cố login vô để comment cho bạn.

**Reply by thanhlmm** [reply]
cảm ơn bạn :D
